Tests/NimbleTests/AsyncAwaitTest.swift

```diff
@@ -189,6 +189,15 @@ final class AsyncAwaitTest: XCTestCase { // swiftlint:disable:this type_body_len
         }
     }
 
+    func testWaitUntilUsingSendable() async {
+        await waitUntil { done in
+            let queue = OperationQueue()
+            let op = BlockOperation {}
+            op.completionBlock = done
```

Comment:
- On Swift5: 
  - Before
    - Warning: `Assigning non-sendable parameter 'done' to a @Sendable closure`
  - After
    - No-Warning